src/System.Private.CoreLib/shared/System/Buffers/Text/FormattingHelpers.CountDigits.cs

```diff
@@ -10,9 +10,40 @@ namespace System.Buffers.Text
 {
     internal static partial class FormattingHelpers
     {
+        private static readonly byte[] s_uInt64MaxLog10GivenLzCount = 
```

Comment:
For these static readonly byte[]s (not the other array types, just byte[]), you can instead do something like this:
https://github.com/dotnet/coreclr/blob/e73c16c3861b071eca336fedaf19326c3e123d97/src/System.Private.CoreLib/shared/System/IO/Path.cs#L663

Reply:
Can you educate me on what the optimization is for byte arrays in this case?

Reply:
https://github.com/dotnet/roslyn/pull/24621